blocks/workspace.js

```diff
@@ -113,83 +113,42 @@ class Workspace extends Newsletter {
     })
   }
 
-  acceptDrop (script, x, y, snapTo, wrappingC) {
+  acceptDrop (script, x, y, snapTo, wrappingC, undoEntry) {
```

Comment:
As in a previous comment, I think most of these aren't needed anymore, but I think `acceptDrop` will be moved for only non-Workspaces anyways